src/main/java/de/dennisguse/opentracks/adapters/AggregatedStatisticsAdapter.java

```diff
@@ -66,6 +69,20 @@ public View getView(int position, View convertView, ViewGroup parent) {
         return convertView;
     }
 
+    public AggregatedStatistics swapData(AggregatedStatistics aggregatedStatistics) {
+        this.aggregatedStatistics = aggregatedStatistics;
+        this.notifyDataSetChanged();
+        return aggregatedStatistics;
+    }
+
+    public List<String> getCategories() {
+        List<String> categories = new ArrayList<>();
+        for (int i = 0; i < aggregatedStatistics.getCount(); i++) {
```

Comment:
I prefer not to do that because It would result in `AggregatedStatistics` modifications. Maybe, creating `getCategories` method inside `AggregatedStatistics`. Would it worth?